syntax = "proto3";
package game;

enum PacketID {
  NONE = 0;
  LOGIN_REQ = 1001;
  LOGIN_RES = 1002;
  LOGOUT_REQ = 1003;
  LOGOUT_RES = 1004;
  GET_CHARACTER_LIST_REQ = 1005;
  GET_CHARACTER_LIST_RES = 1006;
}

message LoginRequest {
  string user_id = 1;
}

message LoginResponse {
  bool success = 1;
  string message = 2;
}

message LogoutRequest {
  string user_id = 1;
}

message GetCharacterListRequest {
  string user_id = 1;
}

message GetCharacterListResponse {
  repeated string characters = 1;
}